cmd/podman/ps.go

```diff
@@ -190,6 +191,7 @@ func psInit(command *cliconfig.PsValues) {
 	flags.BoolVarP(&command.Size, "size", "s", false, "Display the total file sizes")
 	flags.StringVar(&command.Sort, "sort", "created", "Sort output by command, created, id, image, names, runningfor, size, or status")
 	flags.BoolVar(&command.Sync, "sync", false, "Sync container state with OCI runtime")
+	flags.UintVarP(&command.Watch, "watch", "w", 1, "Watch the ps output on a interval in seconds")
```

Comment:
is there a way it can be a decimal?

watch(1) supports intervals specified as decimals, e.g. `watch -n 0.1` will update 10 times per second

Reply:
ignore the previous comment if you want to get this in time for the release